test/version/version.test.js

```diff
@@ -157,17 +156,6 @@ describe('single version flag', () => {
         expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
     });
 
-    it('outputs version with init', () => {
-        const { exitCode, stderr, stdout } = run(__dirname, ['init', '--version'], false);
-
-        expect(exitCode).toBe(0);
-        expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/init ${initPkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
-    });
```

Comment:
Why it was deleted?

Reply:
As init is deleted as of this PR, this tests version of init.

Reply:
So `webpack init --version` isn't supported anymore?

Reply:
Should work, we just move `init` to generators package, `webpack init --version` should still work

Reply:
Hmm, My bad. It should be same as generators' version now. Will add both tests with updates.